opm/simulators/wells/GasLiftSingleWell_impl.hpp

```diff
@@ -112,6 +112,11 @@ computeWellRates_(
             -potentials[this->water_pos_]);
         displayDebugMessage_(msg);
     }
+
+    for (size_t phase = 0; phase < potentials.size(); ++phase){
+        // make sure the potentials are positive
```

Comment:
This comment disagrees with the code so either
1. The comment is wrong
2. The code is wrong
3. Both the code and the comment are wrong

Reply:
The comment is wrong. It should be negative. The sign is changed later.